src/theme/ItaliaTheme/Blocks/_bandiInEvidenceTemplate.scss

```
@@ -50,9 +50,15 @@
   }
 
   // tablet
-  @media (max-width: 1367px) and (min-width: 586px) {
+  @media (min-width: 586px) and (max-width: 1367px) {
     .bandi-in-evidence-cards-wrapper {
       grid-template-columns: auto auto;
+
+      div.read-more a.read-more {
+        display: flex;
+        justify-content: right;
+        flex-direction: row;
```

```suggestion
        flex-direction: row;
```

Le altre righe ci sono già: è già flex ed è già allineato a destra, basta la direction